elastic/integrations
# [Cloud Posture] add benchmark id to rule template

## What does this PR do?

this PR uses the newly introduced `benchmark.id` property and assigns every existing rule templates to use `cis_k8s` as its benchmark id. 
when EKS rule templates are introduced, they should be added with their benchmark id property set to `cis_eks`


## Checklist







## How to test this PR locally

1. use local package registry
   - setup `xpack.fleet.registryUrl: 'http://localhost:8080'` in `kibana.dev.yml` 
   - pull this PR [integrations PR](https://github.com/elastic/integrations/pull/3654) 
   - add an `eks`  rule
      - copy-paste a `cis_k8s` rule
      - change its `id` so it's unique (replace a single char)
      - change the `benchmark.id` to `cis_eks` 
   - run `elastic-package build && elastic-package stack up -v -s package-registry`  
2. add a CIS integration per benchmark and verify the number of rules is `92` for `cis_k8s` and `1` for `cis_eks` 
  
 example:
![Screen Shot 2022-07-11 at 16 06 55](https://user-images.githubusercontent.com/20814186/178271181-e0acfd22-7c73-4765-aff1-0a8b671f1caf.png)
